docs/development.md

```diff
@@ -228,6 +213,47 @@ npm --prefix view test -- --watchAll
 npm --prefix view run cypress:run
 ```
 
+## Adding a new version of LLVM to the project
+
+Diffkemp is based on augmentation of the
+[`FunctionComparator`](https://llvm.org/doxygen/classllvm_1_1FunctionComparator.html);
+however, this class is not polymorphic in the upstream and because of that we
+have to add it to the project manually. The steps required to do so are:
+
+1. Extract the
+   [`FunctionComparator.cpp`](https://github.com/llvm/llvm-project/blob/main/llvm/lib/Transforms/Utils/FunctionComparator.cpp)
+   and
+   [`FunctionComparator.h`](https://github.com/llvm/llvm-project/blob/main/llvm/include/llvm/Transforms/Utils/FunctionComparator.h)
+   from the LLVM project monorepo, take the most recent release of your target
+   major version (i.e., prefer
+   [19.1.7](https://github.com/llvm/llvm-project/releases/tag/llvmorg-19.1.7)
+   over
+   [19.1.6](https://github.com/llvm/llvm-project/releases/tag/llvmorg-19.1.6)).
+2. Create a new subdirectory at path (expecting that you are located in the root
+   of the DiffKemp repo): `./diffkemp/simpll/llvm-lib/<MAJOR VERSION>`, where
```

Comment:
I think there's no need to add that parenthesis and the path can be simpler. I'd go with

```suggestion
2. Create a new subdirectory at path `diffkemp/simpll/llvm-lib/<MAJOR VERSION>`, where
```